tasks/sbd_watchdog.yml

```
@@ -0,0 +1,46 @@
+# SPDX-License-Identifier: MIT
+---
+- name: Configure SBD watchdog
+  block:
+    - name: SBD - Configure and unload watchdog kernel modules from blocklist
+      block:
+        - name: "SBD - Configure watchdog kernel module blocklist"
+          ansible.builtin.lineinfile:
+            path: "/etc/modprobe.d/{{ item }}.conf"
+            create: yes
```

```suggestion
            create: true
```
please use `true`/`false` instead of `yes`/`no` (I know there are other places in the ha_cluster role that use yes/no - but ansible-lint 6.x is much stricter about this)

---

done